lib.ml

```diff
@@ -22,6 +22,7 @@ let alias_env : Types.tycon_environment =
 let datatype = DesugarDatatypes.read ~aliases:alias_env
 
 let cgi_parameters = ref []
```

Comment:
I gather these are among the global references that need to be made thread-local.